src/browser-driver-manager.js

```
@@ -1,259 +1,124 @@
-  console.error(chalk.red('browser-driver-manager error:'), msg);
-  process.exitCode = 1;
-}
-
-/**
- * Output a message for verbose logging
- * @param {Object} chalk
- * @param {Boolean} verbose - If verbose logging is enabled
- * @param {String} msg - Message to display
- */
-async function verboseLog(chalk, verbose, msg) {
-  if (verbose) {
-    console.log(chalk.blue('log:'), msg);
+function getEnv() {
+  const envPath = path.resolve(BDM_CACHE_DIR, '.env');
+  if (!fs.existsSync(envPath)) {
+    return null;
   }
+  const env = fs.readFileSync(envPath, 'utf8');
+  return env;
 }
 
-/**
- * Uppercase first character of a string
- * @param {String} str - String to Titlecase
- */
-function titleCase(str) {
-  return str.charAt(0).toUpperCase() + str.substr(1);
-}
+async function browserDriverManager(args) {
+  if (args[0]?.trim() === 'which') {
+    const env = getEnv();
+    console.log(env);
+    return;
+  }
 
-/**
- * Run the main bash script
- * @param {Object} options
- * @param {Boolean} [options.returnValue] - Return the bash script output rather than display it
- * @param {Boolean} [options.sudo] - Run the command as sudo
- * @param {String[]} options.args - List of script arguments
- * @returns {Promise<String|null>}
- */
-function runBashScript({ returnValue = false, sudo = false, args } = {}) {
-  return new Promise((resolve, reject) => {
-    const lines = [];
-    let lastChunk;
+  if (args[0]?.trim() === 'version') {
+    const pattern = /-(\d+\.\d+\.\d+\.\d+)/;
+    const filePath = getEnv();
+    // Search for the pattern in the file path
+    const match = filePath?.match(pattern);
 
-    const scriptPath = path.join(__dirname, 'index.sh');
-    let child;
-    if (sudo) {
-      child = spawn('sudo', [scriptPath, ...args]);
+    if (match) {
+      const version = match[1];
+      console.log('Version:', version);
     } else {
-      child = spawn(scriptPath, args);
+      console.log('Version not found in the file path.');
```

This else case should never happen so long as they've installed before. For both `which` and `version` we should throw an error for if the user has not installed before and there is no `env` file.